### PR TITLE
Use web-url for CI URL

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

**Comment by mike-sul** [comment]
@doanac This is just a general idea, I see a few options here, not sure which would be the best.

File: client/foundries.go [hunk 1]
```diff
@@ -249,6 +249,20 @@ func NewApiClient(serverUrl string, config Config, caCertPath string) *Api {
 	return &api
 }
 
+func getResponse(resp *[]byte, err error, runName string) (string, error) {
+	if err != nil {
+		return "", err
+	}
+	type PatchResp struct {
+		JobServUrl string `json:"web-url"`
```
Comment (doanac) [inline client/foundries.go:257]: I think this needs to be:
~~~
		JobServUrl string `json:"url"`
		WebUrl string `json:"web-url"`
~~~

Then we return this tuple to the caller.

Comment (mike-sul) [reply client/foundries.go:257]: Do you mean jobserve-url? I thought about it too. But a caller, in this case, is a `fioctl targets <subcommand>` command, the question is what a user can do with the URL that points to  API. Maybe it's useful for the case when fioctl is used programmatically by some other tool.

Comment (doanac) [reply client/foundries.go:257]: isn't the caller is this function printing the value out and then passing it tail function? ie - one should be a web-url and the other should be an API url

Comment (mike-sul) [reply client/foundries.go:257]: yes, I just thought that the web-url like this https://ci.foundries.io/projects/msul-dev01/lmp/builds/439/assemble-system-image/artifacts/console.log is also good enough for the tailing.
Anyway, I think, it makes sense to return both URLs - it is just more flexible and fulfills more use cases.

**Comment by mike-sul** [comment]
@doanac I updated this PR according to the latest discussion about it.

File: subcommands/targets/image.go [hunk 2]
```diff
@@ -27,13 +27,13 @@ func doImage(cmd *cobra.Command, args []string) {
 	inputTarget := args[0]
 	logrus.Debugf("Generating image of Target %s in Factory %s", inputTarget, factory)
 
-	url, err := api.TargetImageCreate(factory, inputTarget)
+	JobServUrl, WebUrl, err := api.TargetImageCreate(factory, inputTarget)
```
Comment (doanac) [inline subcommands/targets/image.go:30]: this looks good. just one nit. Can you make this `jobServUrl, webUrl`. It seems a bit unusual in golang to name a local variable with a capital letter.

Comment (mike-sul) [reply subcommands/targets/image.go:30]: absolutely agree.

Comment (mike-sul) [reply subcommands/targets/image.go]: done

**Pull request merged**